R/modules.R

```diff
@@ -192,39 +37,82 @@ is_arg_used <- function(modules, arg) {
 #'   filters in the listed datasets. `NULL` will hide the filter panel,
 #'   and the keyword `'all'` will show filters of all datasets. `datanames` also determines
 #'   a subset of datasets which are appended to the `data` argument in server function.
-#' @param server_args (named `list`) with additional arguments passed on to the
-#'   server function.
-#' @param ui_args (named `list`) with additional arguments passed on to the
-#'   UI function.
+#' @param server_args (named `list`) with additional arguments passed on to the server function.
+#' @param ui_args (named `list`) with additional arguments passed on to the UI function.
+#' @param ...
```

Comment:
@chlebowa this does not end up in the final documentation since `...` are also documented in `print.teal_module` and this gets over-written. You brought back the issue we tried to fix during https://github.com/insightsengineering/teal/pull/1073

<img width="417" alt="image" src="https://github.com/insightsengineering/teal/assets/133694481/e5cdfc77-0fc7-4cc9-90de-6922cc1420e3">

Reply:
Indeed.

Reply:
af6c569a2cfd978eacaf4a9ee57f7d4bc08b3af9